automake/README.md

```diff
@@ -10,15 +12,21 @@ Binary package
 
 ## Usage
 
-This package provides `automake` the GNU tool to generate make files.
-
-## Usage
-
 Typically this is a build time dependency that can be added to your
 plan.sh:
 
     pkg_build_deps=(core/automake)
 
+## Testing
```

Comment:
Hello! Could you please remove this section from the readme in favor of #2678 ?

Otherwise this looks great and I'd be happy to test and merge it <3

Reply:
Thanks!